src/main/kotlin/no/nav/medlemskap/regler/v1/ReglerForArbeidsforhold.kt

```diff
@@ -63,71 +63,73 @@ class ReglerForArbeidsforhold(val personfakta: Personfakta) : Regler() {
+    private val harBrukerEttArbeidsforhold = Regel(
             identifikator = "ARB-1",
-            avklaring = "Har bruker et registrert arbeidsforhold?",
+            avklaring = "Har bruker hatt et arbeidsforhold i Aa-registeret de siste 12 månedene?",
             beskrivelse = "",
             operasjon = { sjekkArbeidsforhold() }
     )
 
-    private val erArbeidsgiverNorsk = Regel(
+    private val erArbeidsgiverOrganisasjon = Regel(
             identifikator = "ARB-2",
-            avklaring = "Jobber bruker for en norsk arbeidsgiver?",
+            avklaring = "Er foretaket registrert i foretaksregisteret?",
             beskrivelse = "",
             operasjon = { sjekkArbeidsgiver() }
     )
 
     private val harForetakMerEnn5Ansatte = Regel(
-            identifikator = "ARB-6",
-            avklaring = "Er foretaket et reelt foretak som har reell økonomisk aktivitet i Norge?",
+            identifikator = "ARB-3",
+            avklaring = "Har arbeidsgiver sin hovedaktivitet i Norge?",
             beskrivelse = "",
             operasjon = { sjekkOmForetakMerEnn5Ansatte() }
     )
 
     private val erForetakAktivt = Regel(
-            identifikator = "ARB-7",
+            identifikator = "ARB-4",
             avklaring = "Er foretaket aktivt?",
             beskrivelse = "",
             operasjon = { sjekKonkursstatus() }
     )
 
-    private val erBrukerPilotEllerKabinansatt = Regel(
-            identifikator = "ARB-3",
-            avklaring = "Er bruker pilot eller kabinansatt?",
-            beskrivelse = "",
-            operasjon = { sjekkYrkeskodeLuftfart() }
-    )
-
     private val erArbeidsforholdetMaritimt = Regel(
-            identifikator = "ARB-4",
+            identifikator = "ARB-5",
             avklaring = "Har bruker et maritimt arbeidsforhold?",
             beskrivelse = "",
             operasjon = { sjekkMaritim() }
     )
 
     private val jobberBrukerPåNorskSkip = Regel(
-            identifikator = "ARB-5",
+            identifikator = "ARB-6",
             avklaring = "Jobber bruker på et norskregistrert skip?",
             beskrivelse = "",
             operasjon = { sjekkSkipsregister() }
     )
 
+    private val erBrukerPilotEllerKabinansatt = Regel(
+            identifikator = "ARB-7",
+            avklaring = "Er bruker pilot eller kabinansatt?",
+            beskrivelse = "",
+            operasjon = { sjekkYrkeskodeLuftfart() }
+    )
+
     private val yrkeskoderLuftfart = listOf("3143107", "5111105", "5111117")
 
-    private fun sjekkArbeidsforhold(): Resultat =
-            when {
-                personfakta.arbeidsforhold() kunEr 1 -> ja()
-                else -> nei()
-            }
+    private fun sjekkArbeidsforhold(): Resultat {
+
+        if (!personfakta.harSammenhengendeArbeidsforholdSiste12Mnd())
+            return nei("Arbeidstaker har ikke sammenhengende arbeidsforhold siste 12 mnd")
+
+        if (personfakta.arbeidsforhold() antallErIkke 1)
```

Comment:
Kan man ikke ha mer enn ett arbeidsforhold?

Reply:
Nei, men det gjelder kun siste 28 dager, som er det metoden henter ut. Kanskje personfakta.arbeidsforhold() burde endres navn til f.eks. arbeidsforholdIOpptjeningsperiode() for å gjøre det klarere.

Reply:
Men hvis man i den perioden har mer enn ett arbeidsforhold, så får man ikke medlemskap? Det virker litt merkelig..?